### PR TITLE
EVG-7755 add more logging to host allocaator

### PR DESCRIPTION
**Pull request by john-m-liu**
We already figured out the problem but I think this is still helpful

### CODE REVIEW

File: scheduler/utilization_based_host_allocator.go [hunk 1]
```diff
@@ -91,6 +92,21 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAl
 	return numNewHostsToRequest, nil
 }
 
+func logInput(data HostAllocatorData) {
+	existingHosts := []string{}
+	for _, h := range data.ExistingHosts {
+		existingHosts = append(existingHosts, h.Id)
+	}
+	grip.Info(message.Fields{
```
Comment (tychoish) [inline scheduler/utilization_based_host_allocator.go:100]: put "runner": RunnerName

File: scheduler/utilization_based_host_allocator.go [hunk 2]
```diff
@@ -91,6 +92,21 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAl
...
+		"message":            "utilization-based host allocator input",
+		"distro":             data.Distro.Id,
+		"existing_hosts":     existingHosts,
```
Comment (tychoish) [inline scheduler/utilization_based_host_allocator.go:103]: maybe omit this?

Comment (tychoish) [reply scheduler/utilization_based_host_allocator.go:103]: or just have a number?

Comment (john-m-liu) [reply scheduler/utilization_based_host_allocator.go]: made it just the count

File: scheduler/utilization_based_host_allocator.go (hunk 1 above)
Comment (john-m-liu) [reply scheduler/utilization_based_host_allocator.go]: done

**Pull request merged**